src/sentry/services/hybrid_cloud/project/impl.py

```diff
@@ -113,3 +114,29 @@ def create_project_for_organization(
             )
 
             return serialize_project(project)
+
+    def get_or_create_project_for_organization(
+        self,
+        *,
+        organization_id: int,
+        project_name: str,
+        platform: str,
+        user_id: int,
+        add_org_default_team: Optional[bool] = False,
+    ) -> RpcProject:
+        project_query = Project.objects.filter(
```

Comment:
Do we need stable ordering for this query as well? I don't think this guarantees we select the same project every time if there is more than one

Reply:
hm! That's a good point. Let me fetch by slug to make sure it's unique.

Reply:
I looked into using slug, but if I generate the slug for new projects here it causes inconsistency with how the project model generates slug. Will go with having this list ordered. It doesn't matter in the logic which one we return but let's have it consistent.